rllib/policy/eager_tf_policy_v2.py

```diff
@@ -483,14 +483,20 @@ def compute_actions_from_input_dict(
                 timestep=timestep, explore=explore, tf_sess=self.get_session()
             )
 
-        ret = self._compute_actions_helper(
-            input_dict,
-            state_batches,
-            # TODO: Passing episodes into a traced method does not work.
-            None if self.config["eager_tracing"] else episodes,
-            explore,
-            timestep,
-        )
+        if self.config.get("_enable_rl_module_api"):
+            if explore:
+                ret = self._compute_actions_helper_rl_module_explore(input_dict)
+            else:
+                ret = self._compute_actions_helper_rl_module_inference(input_dict)
+        else:
+            ret = self._compute_actions_helper(
+                input_dict,
```

Comment:
no I meant instead of positional args
```
ret = self._compute_actions_helper(
                input_dict,
                state_batches,
                # TODO: Passing episodes into a traced method does not work.
                None if self.config["eager_tracing"] else episodes,
                explore,
                timestep,
            )
```

use key word args

```
ret = self._compute_actions_helper(
                input_dict=input_dict,
                state=state_batches,
                # TODO: Passing episodes into a traced method does not work.
                eager_tracing=None if self.config["eager_tracing"] else episodes,
                explore=explore,
                timestep=timestep,
            )
```

Reply:
Ahh, you mean "named args", got it! :D I was thinking using actual `**kwargs`.

Yeah, definitely, we should actually always(!) use named args, even force this via the asterisk (*) at the beginning of the signature.